docs/docs/dev_docs/tutorials/token_portal/withdrawing_to_l1.md

```diff
@@ -42,8 +42,10 @@ For both the public and private flow, we use the same mechanism to determine the
 After the transaction is completed on L2, the portal must call the outbox to successfully transfer funds to the user on L1. Like with deposits, things can be complex here. For example, what happens if the transaction was done on L2 to burn tokens but can’t be withdrawn to L1? Then the funds are lost forever! How do we prevent this?
 
 Paste this in your `TokenPortal.sol`:
-
-#include_code token_portal_withdraw /l1-contracts/test/portals/TokenPortal.sol solidity
+```solidity
```

Comment:
Why this change? Running the docs locally, this removes the link to source at the bottom of the code snippet. Is it just for the closing curly brace? If so, I agree with the change. 

Do you think we should add a link to the source below the reference?

Reply:
Ok that makes sense. If there are links to source elsewhere, not a big deal.